src/request/sign-transaction/SignTransaction.js

```diff
@@ -178,14 +216,14 @@ class BaseLayout {
     }
 
     run() {
-        // Go to start page
-        window.location.hash = BaseLayout.Pages.CONFIRM_TRANSACTION;
-
         // Async pre-load the crypto worker to reduce wait time at first decrypt attempt
         Nimiq.CryptoWorker.getInstanceAsync();
-        /** @type {HTMLElement} */
-        (this.$el.parentElement).classList.remove('display-none');
-        this._passphraseBox.focus();
+        // Go to start page
+        window.location.hash = SignTransaction.Pages.CONFIRM_TRANSACTION;
+        const width = window.innerWidth
+                || document.documentElement.clientWidth
+                || document.getElementsByTagName('body')[0].clientWidth;
+        if (width > 600) this._passphraseBox.focus();
```

Comment:
The `600` will be used in other requests as well, right? It could thus be moved to a constants collection. Please add a TODO for it.

Reply:
Yes, the constant collection is something that I have in mind for that as well.